lib/id3c/cli/redcap.py

```diff
@@ -795,3 +802,13 @@ def __str__(self):
         # to see things exactly as they were, with minimal post-processing,
         # when troubleshooting.
         return f"{self.response.status_code} {self.response.reason}: {self.response.text}"
+
+
+class APIJsonDecodeError(json.decoder.JSONDecodeError):
```

Comment:
I'd suggest matching our name (and casing) of the exception with the stdlib, using just `class JSONDecodeError` here. Any future catching of it can be scoped by module (e.g. `id3c.cli.redcap.JSONDecodeError`). This also suggests to me that maybe we should add this "enhanced" `JSONDecodeError` to the `id3c.json` library instead…